cli.js

```diff
@@ -67,7 +72,23 @@ process.stdin.on('keypress', (ch, key) => {
 		query.pop();
 	} else if (key.ctrl && key.name === 'u') {
 		query.length = 0;
-	} else if (key.name === 'return' && prevResult) {
+	} else if (key.ctrl && key.name === '0') {
+		npc.copy(prevResult);
+	} else if (key.ctrl && key.name === '1') {
```

Comment:
@sindresorhus yeah, I'm still working on it, was doing some testing yesterday. I don't think the ctrl + number combination is gonna work (on macOS at least). The combination is already assigned or something.